bin/bff.R

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env Rscript
+
+if (!require("BiocManager", quietly = TRUE))
+    install.packages("BiocManager",repos = "http://cran.us.r-project.org")
+is_demuxmix<-require("demuxmix")
+if (!require("demuxmix", quietly = TRUE))
+    BiocManager::install("demuxmix")
+  BiocManager::install("demuxmix")
+if (!require("DropletUtils", quietly = TRUE))
+  BiocManager::install("DropletUtils")
+if (!require("preprocessCore", quietly = TRUE))
+  BiocManager::install("preprocessCore")
+if (!require("cellhashR", quietly = TRUE))
+  devtools::install_github(repo = 'bimberlab/cellhashR', ref = 'master', dependencies = TRUE, upgrade = 'always')
+
```

Comment:
well.. we need to find a better solution for that

Reply:
This tools is not compatible with conda on my computer. I already asked for someone else to test and add what is necessary for this script. I cannot run it on the pipeline - arm64 incompatible